modules/iam-assumable-role-with-oidc/main.tf

```diff
@@ -75,7 +75,7 @@ data "aws_iam_policy_document" "assume_role_with_oidc" {
         for_each = length(var.oidc_fully_qualified_audiences) > 0 ? local.urls : []
 
         content {
-          test     = "StringLike"
+          test     = "StringEquals"
```

Comment:
I think this is better suited to be a variable if we need to change it - see https://github.com/terraform-aws-modules/terraform-aws-iam/blob/3ec0f0fd5d360019231f14fc7431418b52deda0c/modules/iam-role-for-service-accounts-eks/variables.tf#L76

Reply:
Fixed